### PR TITLE
studio: doc 1. show, hide commits; 2. static images

### PR DESCRIPTION
**Pull request by tapadipti**
- Described show/hide commits.
- Moved pages related to views inside the views menu.

### CODE REVIEW

**Comment by jorgeorpinel** [comment]
Long PR title 😄

File: content/docs/studio/user-guide/views/explore-experiments.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
+# Explore ML Experiments
+
+When you create a view by connecting to a Git repository, the view is added to
+your DVC Studio dashboard. You can dive deep into the views shown in your DVC
```
Comment (jorgeorpinel) [inline content/docs/studio/user-guide/views/explore-experiments.md:4]: Did anything change from https://dvc.org/doc/studio/user-guide/explore-experiments ? Git didn't catch the move, instead it shows content/docs/studio/user-guide/explore-experiments.md deleted and this one crated.

**Comment by jorgeorpinel** [comment]
> Note: @tapadipti I assigned you so we can see your avatar in https://github.com/orgs/iterative/projects/238, where I added this. Standard #docs team practice 😬

File: content/docs/studio/user-guide/views/explore-experiments.md (hunk 1 above)
Comment (jorgeorpinel) [reply content/docs/studio/user-guide/views/explore-experiments.md:4]: Ok yeah I see it's different from https://dvc-org-hide-commits-8bc2l1z20.herokuapp.com/doc/studio/user-guide/views/explore-experiments . A summary of changes would be really useful for the review but I'll compare the deployed versions if not. Thanks

File: content/docs/sidebar.json [hunk 2]
```diff
@@ -562,7 +562,7 @@
             "slug": "prepare-repositories"
           },
           {
-            "label": "Views",
+            "label": "Views and experiments",
```
Comment (jorgeorpinel) [inline content/docs/sidebar.json:565]:
```suggestion
            "label": "Views and Experiments",
```

File: content/docs/studio/user-guide/views/explore-experiments.md (hunk 1 above)
Comment (tapadipti) [reply content/docs/studio/user-guide/views/explore-experiments.md:4]: @jorgeorpinel Here are the changes in that file:

- Added a table of contents (3 items) in the `Components of a view` section
- Updated the gif for the `Display preferences` section. But the file was updated in the `static` repo, so there are no code changes here.
- Changed the bullet points within `Display preferences` into sub-headings (####). Eg, `Filters`, `Columns`, etc.
- Added sub-section `Hide commits` within `Display preferences`.

File: content/docs/sidebar.json [hunk 3]
```diff
@@ -562,7 +562,7 @@
             "slug": "prepare-repositories"
           },
           {
-            "label": "Views",
+            "label": "Views and Experiments",
```
Comment (jorgeorpinel) [inline content/docs/sidebar.json:565]: Q: So why do we want to bundle Views and Experiments topics in a single section? Maybe create an **Experiments** section instead to contain Explore ML Experiments, Visualize and Compare, and Run Experiments?

File: content/docs/studio/troubleshooting.md [hunk 4]
```diff
@@ -147,6 +148,20 @@ view. Once you show/hide columns, you can save the changes. Check out the
 [above video](#showhide-columns) to see how you can show/hide columns. Once you
 show/hide columns, remember to save the changes.
 
+## View does not contain some of my commits or branches
+
+This is likely not an error. DVC Studio identifies commits that do not change
+metrics and will auto-hide such non-experiment commits. You can also manually
```
Comment (jorgeorpinel) [inline content/docs/studio/troubleshooting.md:154]: We don't [explain experiments](https://dvc.org/doc/user-guide/experiment-management/experiments-overview) as commits. In fact technically they are not commits but custom Git references. So hopefully there's no need for this clarification.

```suggestion
metrics and will auto-hide such commits. You can also manually
```

Comment (jorgeorpinel) [reply content/docs/studio/troubleshooting.md:154]: Is this a common phrase in Studio docs though?

File: content/docs/sidebar.json (hunk 3 above)
Comment (tapadipti) [reply content/docs/sidebar.json:565]: I added it within the same menu coz everything you do in terms of experiments is within the view. Exploring experiments is the same as exploring a view. So, couldn't draw a clear boundary between views and experiments. This is different from `Teams`, `Accounts` etc which are clearly different. But let me know if it's better to separate them and I can move the pages as you have suggested.

File: content/docs/studio/troubleshooting.md [hunk 5]
```diff
@@ -147,6 +148,20 @@ view. Once you show/hide columns, you can save the changes. Check out the
...
+hide commits and branches. So, it is possible that the commits or branches you
+do not see in your view were manually hidden by you or someone else in your
+team.
+
+You can unhide hidden commits to display them in your view. For details, refer
+[Display preferences -> Hide commits](/doc/studio/user-guide/views/explore-experiments#hide-commits).
```
Comment (jorgeorpinel) [inline content/docs/studio/troubleshooting.md:160]:
```suggestion
You can unhide hidden commits to display them in your view. For details, refer to
[Display preferences -> Hide commits](/doc/studio/user-guide/views/explore-experiments#hide-commits).
```

File: content/docs/studio/troubleshooting.md (hunk 4 above)
Comment (tapadipti) [reply content/docs/studio/troubleshooting.md:154]: No, it's not. We just started using this term for commits that are likely not experimentation attempts. Eg, if someone added a readme file, or if they restructured their repo, or did anything else that is not about updating params, files, metrics etc., then all those commits are non-experiments. I used the same term in the last Studio product update email. But if you have a suggestion for a different way of writing this, do suggest.

File: content/docs/studio/troubleshooting.md [hunk 6]
```diff
@@ -147,6 +148,20 @@ view. Once you show/hide columns, you can save the changes. Check out the
...
+
+However, if the commit/branch that is missing is not in the hidden commits list
+also, then please [raise a support request](#support).
```
Comment (jorgeorpinel) [inline content/docs/studio/troubleshooting.md:163]:
```suggestion
However, if the missing commit/branch is not in the hidden commits list, then
please [raise a support request](#support).
```

File: content/docs/studio/user-guide/views/explore-experiments.md (hunk 1 above)
Comment (jorgeorpinel) [reply content/docs/studio/user-guide/views/explore-experiments.md:4]: Thanks, will review this tomorrow (wrapping up for today!)

File: content/docs/sidebar.json (hunk 3 above)
Comment (jorgeorpinel) [reply content/docs/sidebar.json:565]: Up to you on this. I'm not familiar enough with this region of the docs still, unfort.

File: content/docs/studio/troubleshooting.md (hunk 4 above)
Comment (tapadipti) [reply content/docs/studio/troubleshooting.md:154]: If we change this here, we also need to change it in `explore-experiments.md`; coz there too, I've used this term. Does it make sense to call them `noisy` commits or `noise`? Or better to stick with just `.. such commits`?

Comment (jorgeorpinel) [reply content/docs/studio/troubleshooting.md:154]: > Eg, if someone ... restructured their repo, or did anything else that is not about updating params, files, metrics etc., then all those commits are non-experiments

Oh. I see. So we're not even talking about our special "DVC Experiments" (`dvc exp run`) here. This can be confusing, let me think 🤔

Comment (tapadipti) [reply content/docs/studio/troubleshooting.md:154]: yes, this isn't about `exp`. I think it's okay to just call them `such commits`, meaning, commits that don't change metrics and files.

File: content/docs/studio/user-guide/views/visualize-experiments.md [hunk 7]
```diff
@@ -1,22 +1,66 @@
...
+
+1. Data series files, which can be JSON, YAML, CSV or TSV. Data from these files
+   will populate your AUC curves, loss functions and other metric plots.
+2. Image files in JPEG, GIF, or PNG format. These images will be displayed
+   directly in DVC Studio.
+
+You can define multiple plots in a single repository. Below is an example
+snippet from a `dvc.yaml` file showing the `evaluate` stage of the dvc pipeline.
+
+```
```
Comment (shcheklein) [inline content/docs/studio/user-guide/views/visualize-experiments.md:28]: add `yaml` after the tripple "\`"

File: content/docs/studio/user-guide/views/visualize-experiments.md [hunk 8]
```diff
@@ -1,22 +1,66 @@
...
-one experiment, you can use the plots to compare them.
+DVC Studio can work with two types of plots files in your repository:
+
+1. Data series files, which can be JSON, YAML, CSV or TSV. Data from these files
+   will populate your AUC curves, loss functions and other metric plots.
+2. Image files in JPEG, GIF, or PNG format. These images will be displayed
+   directly in DVC Studio.
+
+You can define multiple plots in a single repository. Below is an example
+snippet from a `dvc.yaml` file showing the `evaluate` stage of the dvc pipeline.
```
Comment (shcheklein) [inline content/docs/studio/user-guide/views/visualize-experiments.md:26]: dvc -> DVC

**Comment by shcheklein** [comment]
@jorgeorpinel hey, are we done with the review here?

**Comment by jorgeorpinel** [comment]
Will check today, sorry for the delay! ⌛

File: content/docs/studio/troubleshooting.md [hunk 9]
```diff
@@ -147,6 +148,19 @@ view. Once you show/hide columns, you can save the changes. Check out the
...
 
+## View does not contain some of my commits or branches
+
+This is likely not an error. DVC Studio identifies commits that do not change
+metrics and will auto-hide such commits. You can also manually hide commits and
+branches. So, it is possible that the commits or branches you do not see in your
+view were manually hidden by you or someone else in your team.
+
+You can unhide hidden commits/branches to display them in your view. For
+details, refer to
```
Comment (jorgeorpinel) [inline content/docs/studio/troubleshooting.md:159]: Small copy edit (committing):
```suggestion
You can unhide commits/branches to display them in your view. For details,
refer to
```

File: content/docs/studio/user-guide/views/explore-experiments.md [hunk 10]
```diff
@@ -0,0 +1,116 @@
...
+
+#### **Selected only:**
+
+Use this toggle switch to show/hide experiments that you have not selected.
+
+#### **Delta mode:**
+
+Toggle between absolute values and difference from the first row.
+
+#### **Save changes:**
```
[the remark this answers is not in the document]
Comment (jorgeorpinel) [reply content/docs/studio/user-guide/views/explore-experiments.md:98]: Do we need headers for single sentences?

File: content/docs/studio/user-guide/views/explore-experiments.md [hunk 11]
```diff
@@ -0,0 +1,116 @@
...
+- [View contains columns that I did not import](/doc/studio/troubleshooting#view-contains-columns-that-i-did-not-import)
+
+#### **Hide commits:**
+
+The following functionality are available for you to hide irrelevant commits
+from your views in DVC Studio.
+
+- **Hide commits and branches manually:** You can hide irrelevant commits, and
+  even entire branches. This means that commits you made simply to re-organize
+  your project structure will not take up space in your view anymore. To hide a
```
Comment (jorgeorpinel) [inline content/docs/studio/user-guide/views/explore-experiments.md:78]: > This means that commits you made simply to re-organize your project...

Quite complicated sentence, not easy to read and comprehend. Can this motivation be simplified and merged with the previous sentence?

p.s. avoid "simply" as rule of thumb.

File: content/docs/studio/user-guide/views/explore-experiments.md [hunk 12]
```diff
@@ -0,0 +1,116 @@
...
+  commit or branch, click on the 3-dot menu next to the commit or branch name
+  and click on `Hide commit` (or `Hide branch`).
```
Comment (jorgeorpinel) [inline content/docs/studio/user-guide/views/explore-experiments.md:80]: 💅 
```suggestion
  and click on `Hide commit` or `Hide branch`.
```

File: content/docs/studio/user-guide/views/explore-experiments.md [hunk 13]
```diff
@@ -0,0 +1,116 @@
...
+- **DVC Studio auto-hides irrelevant commits:** DVC Studio identifies commits
+  that do not change metrics and will auto-hide such commits.
```
Comment (jorgeorpinel) [inline content/docs/studio/user-guide/views/explore-experiments.md:82]: I'd put this bullet first since it happens automatically.

And 💅 
```suggestion
- **DVC Studio auto-hides irrelevant commits:** DVC Studio identifies commits
  where metrics did not change and hides them automatically.
```

**Review by jorgeorpinel — COMMENTED** [review]
only minor things so far 👍

File: content/docs/studio/user-guide/views/visualize-experiments.md (hunk 7 above)
Comment (jorgeorpinel) [reply content/docs/studio/user-guide/views/visualize-experiments.md:28]: (committing)
````suggestion
```yaml
````

File: content/docs/studio/user-guide/views/visualize-experiments.md (hunk 8 above)
Comment (jorgeorpinel) [reply content/docs/studio/user-guide/views/visualize-experiments.md:26]:
```suggestion
snippet from a `dvc.yaml` file showing the `evaluate` stage of the DVC pipeline.
```

File: content/docs/studio/troubleshooting.md (hunk 4 above)
Comment (jorgeorpinel) [reply content/docs/studio/troubleshooting.md]: I think we should just go with

"commits where metrics did not change and will hide them automatically"

Don't mix the concept of experimentation here 🙂

**Comment by jorgeorpinel** [comment]
Other than minor things and formatting this is in great shape, thanks @tapadipti !

File: content/docs/studio/troubleshooting.md (hunk 4 above)
Comment (tapadipti) [reply content/docs/studio/troubleshooting.md]: Yes, I removed term `non-experiments`.

File: content/docs/studio/user-guide/views/explore-experiments.md [hunk 14]
```diff
@@ -0,0 +1,116 @@
...
+
+## Components of a view
+
+The tabular display has different components that show you the complete
+experiment history as well as enable to you to generate plots, compare
+experiments, run new experiments, etc. The major components of a view are:
+
+- [Git history, which represents your experimentation history](#git-history)
+- [Display preferences to show/hide branches, commits and columns, and re-arrange your view](#display-preferences)
+- [Buttons to visualize, compare and run experiments](#visualize-compare-and-run-experiments)
```
[the remark this answers is not in the document]
Comment (tapadipti) [reply content/docs/studio/user-guide/views/explore-experiments.md]: Done.

File: content/docs/studio/user-guide/views/explore-experiments.md [hunk 15]
```diff
@@ -0,0 +1,116 @@
...
+![](https://static.iterative.ai/img/studio/view_components_1.gif)
+
+### Display preferences
+
+The table contains buttons to specify filters and other preferences regarding
+which commits and columns to display.
+
+![](https://static.iterative.ai/img/studio/view_components_2.gif)
+
+#### **Filters:**
```
[the remark this answers is not in the document]
Comment (tapadipti) [reply content/docs/studio/user-guide/views/explore-experiments.md]: Yes, there are too many header levels. I had to add them so that I can link to these sections. But maybe can revise the structure later.

File: content/docs/studio/user-guide/views/explore-experiments.md [hunk 16]
```diff
@@ -0,0 +1,116 @@
...
+  those experiments for which the value of `avg_prec` is greater than `0.9`.
+- **Metric delta:** Change in the value of the metric. For instance, you can use
+  this filter to only display those experiments for which the value of
+  `avg_prec` changed by more than `0.1` compared to the baseline experiment.
+- **Param:** Values of different parameters
+- **File size:** Size of the data, model and other files corresponding to your
+  experiments
+- **File changed:** Whether or not any given file changed in the experiment
+
+#### **Columns:**
```
[the remark this answers is not in the document]
Comment (tapadipti) [reply content/docs/studio/user-guide/views/explore-experiments.md]: Done

File: content/docs/studio/user-guide/views/explore-experiments.md [hunk 17]
```diff
@@ -0,0 +1,116 @@
...
+as per your preferences.
+
+If your view is missing some required columns or includes columns that you do
+not want, refer to the following troubleshooting sections to understand why this
+may have happened.
+
+- [View does not contain the columns that I want](/doc/studio/troubleshooting#view-does-not-contain-the-columns-that-i-want)
+- [View contains columns that I did not import](/doc/studio/troubleshooting#view-contains-columns-that-i-did-not-import)
+
+#### **Hide commits:**
```
[the remark this answers is not in the document]
Comment (tapadipti) [reply content/docs/studio/user-guide/views/explore-experiments.md]: Done

File: content/docs/studio/user-guide/views/explore-experiments.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (tapadipti) [reply content/docs/studio/user-guide/views/explore-experiments.md]: I had to make them headers to keep them consistent with the other sections which are not single sentences. But maybe we can revise the overall structure later.

File: content/docs/studio/user-guide/views/explore-experiments.md (hunk 11 above)
Comment (tapadipti) [reply content/docs/studio/user-guide/views/explore-experiments.md]: Changed

File: content/docs/studio/user-guide/views/explore-experiments.md (hunk 12 above)
Comment (tapadipti) [reply content/docs/studio/user-guide/views/explore-experiments.md]: Done

File: content/docs/studio/user-guide/views/explore-experiments.md (hunk 13 above)
Comment (tapadipti) [reply content/docs/studio/user-guide/views/explore-experiments.md]: Changed (and also mentioned "files and hyperparameters").

**Review by shcheklein — APPROVED** [review]
